include/bitcoin/bitcoin/config/script.hpp

```diff
@@ -83,6 +83,13 @@ class BC_API script
      */
     const std::string to_string() const;
 
+    /**
+     * Return a pretty-printed copy of the script.
+     * @param[in]  flags  The rule_fork flags to use.
+     * @return  A mnemonic-printed copy of the internal script.
+     */
+    const std::string to_string(uint32_t) const;
```

Comment:
missing parameter name. why not use a default parameter (machine::rule_fork::all_rules) on a single method?

Reply:
Of course, that's better :+1:

Updated.